File: Java/OOP/Inheritance/Single inehritance (example 2).java
class Teacher {
    String teacher_name = "James Gosling";
    String subject = "Java Programming";

    void teacherInfo() {
        System.out.println("Teacher Name: " + teacher_name);
    }
}

class Student extends Teacher {
    String student_name = "Abdullah Al Naiem";
    int id = 123456;

    void studentInfo() {
        System.out.println("Student Name: " + student_name);
        System.out.println("Student ID  : " + id);
        System.out.println("Class of    : " + subject);
    }
}

class singleInhertance {
    public static void main(String[] args) {
        Student get = new Student();
        get.teacherInfo();
        get.studentInfo();
    }
}

File: Python/OOP/Inheritance/Single inehritance (example 2).py
class Teacher:
    teacher_name = "Guido van Rossum"
    subject = "Python Programming"

    def teacherInfo(self):
        print("Teacher Name:", self.teacher_name)


class Student(Teacher):
    student_name = "Abdullah Al Naiem"
    id = 123456

    def studentInfo(self):
        print("Student Name:", self.student_name)
        print("Student ID  :", self.id)
        print("Class of    :", self.subject)


class singleInheritance:
    get = Student()
    get.teacherInfo()
    get.studentInfo()